oaknational/Oak-Web-Application
# [PUPIL-864] Add quiz hints to quizzes

## Description

Music year: 2019

Adds hints to quizzes.

## How to test

1. Go to https://deploy-preview-2615--oak-web-application.netlify.thenational.academy/pupils/programmes/maths-primary-year-4/units/calculation-and-conversion-of-measures/lessons/use-efficient-strategies-and-common-measure-conversions-to-solve-problems-in-a-range-of-contexts/starter-quiz
2. Click on the hint in the bottom left
3. You should see a hint and be able to close it.

## Screenshots

<img width="457" alt="image" src="https://github.com/user-attachments/assets/898e5f31-616a-4c83-80f1-6ef391442e79">

## Checklist
